src/main/java/fi/metatavu/dcfb/server/rest/AbstractApi.java

```diff
@@ -122,6 +123,30 @@ protected String slugify(String text) {
     return slugify.slugify(text);
   }
 
+  /**
+   * Parses CSV enum parameter from string list into enum list
+   * 
+   * @param enumType target enum class
+   * @param parameters string values
+   * @return list of enums
+   * @throws IllegalArgumentException if parameters contain invalid values
+   */
+  protected <T extends Enum<T>> List<T> getEnumListParameter(Class<T> enumType, List<String> parameter) {
+    if (parameter == null) {
+      return null;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Return an empty collection instead of null. ([squid:S1168](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1168))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=3&issues=AWQWmPPQbXXTsx7O6YI6&open=AWQWmPPQbXXTsx7O6YI6)

[ISSUE_KEY]: AWQWmPPQbXXTsx7O6YI6